test_util/aws.py

```diff
@@ -33,6 +47,16 @@ def instances_to_hosts(instances):
     return [Host(i.private_ip_address, i.public_ip_address) for i in instances]
 
 
+def fetch_stack(stack_name, boto_wrapper):
+    stack = boto_wrapper.resource('cloudformation').Stack(stack_name)
+    for resource in stack.resource_summaries.all():
+        if resource.logical_resource_id == 'MasterStack':
+            return DcosZenCfStack(stack_name, boto_wrapper)
+        if resource.logical_resource_id == 'MasterServerGroup':
```

Comment:
What's the relationship between `MasterStack` and `DcosZenCfStack` and correspondingly, what's the relationship between `MasterServerGroup` and `DcosCfStack` ?